static/bidder-params/rtbhouse.json

```diff
@@ -7,7 +7,19 @@
         "publisherId": {
             "type": "string",
             "description": "The publisher’s ID provided by RTB House"
+        },
+        "region": {
+            "type": "string",
+            "description": "The publisher’s assigned datacenter region"
+        },
```

Comment:
Region is anti-pattern and I do not recommend making it required. The Prebid Server Committee has discussed this  use case and identified the best approach is for bidders to make their regional endpoints known to hosts for hosts to configure in their data centers as appropriate. This should be an ask for the publisher and become a burden depending on the amount of placements / configurations a publisher needs.

Reply:
@SyntaxNode Thanks for the feedback, we will set this as optional - this is part of our internal process and usually agreed with partner on integration start and is universal, so we don't expect much work from their end in that regard.